data/icons/fluent-icon-theme.index.theme

```diff
@@ -0,0 +1,25 @@
+[Icon Theme]
+Name=Fluent
+Comment=Embedded in qrc to get size dirs matching hicolor theme
+Inherits=hicolor
+Directories=16x16/actions,16x16/places,16x16/status,32x32/categories
+
+[16x16/actions]
+Context=Actions
+Size=16
+Type=Fixed
+
+[16x16/places]
+Context=Places
+Size=16
+Type=Fixed
+
+[16x16/status]
+Context=Status
+Size=16
+Type=Fixed
+
+[32x32/categories]
+Context=Applications
+Size=32
+Type=Fixed
```

Comment:
Checked, the issue was in folders. We can remove all other info, only `Size` is needed.

Reply:
Not sure if I understood what you mean. Only e.g.
```
[16x16/status]
Size=16
```
is needed?

Reply:
Yep. For each category.

Reply:
Hmm, without the `Context` property, declaring 2nd-level directories seems pointless. Are you sure `Context` isn't needed?

Reply:
Yep.

Reply:
This is how it was you pushed the first time, but with correct folder names.

Reply:
Oh well, removed all properties except `Size`